backend/molgenis-emx2-webapi/src/main/java/org/molgenis/emx2/RunMolgenisEmx2.java

```diff
@@ -33,8 +33,7 @@ public static void main(String[] args) {
       db.becomeAdmin();
       if (db.getSchema("pet store") == null) {
         Schema schema = db.createSchema("pet store");
-        PetStoreExample.create(schema.getMetadata());
-        PetStoreExample.populate(schema);
+        new PetStoreLoader().load(schema, true);
```

Comment:
what does true do , maybe use variable name to make clear what the property is meant for